src/main/resources/swagger-ui/yaml/repository.yaml

```diff
@@ -1,9 +1,17 @@
 openapi: "3.0.0"
```

Comment:
also, as we discussed: let's create separate yamles for org (requests with usernames) and flat (without usernames) layout, as users should not see requests they are not allowed to perform in swagger.

Reply:
@olenagerasimova, @dgarus Please review